assets/app/schema.json

```diff
@@ -353,11 +365,161 @@
 							},
 							"children": {
 								"$ref": "#/definitions/driverSettings"
+							},
+							"zwave": {
+								"$ref": "#/definitions/zwaveSetting"
 							}
-						}					
-					}				
+						}
+					}
 				]
 			}
+		},
+		"zwaveDevice": {
+			"type": "object",
+			"required": ["manufacturerId", "productTypeId", "productId"],
+			"properties": {
+				"manufacturerId": {
+					"type": "number"
+				},
+				"productTypeId": {
+					"oneOf": [
+						{
+							"type": "number"
+						},
+						{
+							"type": "array",
+							"items": {
+								"type": "number"
+							}
+						}
+					]
+				},
+				"productId": {
+					"oneOf": [
+						{
+							"type": "number"
+						},
+						{
+							"type": "array",
+							"items": {
+								"type": "number"
+							}
+						}
+					]
+				},
+				"learnmode": {
+					"type": "object",
+					"required": ["instruction"],
+					"properties": {
+						"image": {
+							"type": "string"
+						},
+						"instruction": {
+							"$ref": "#/definitions/i18nObject"
+						}
+					}
+				},
+				"associationGroups": {
+					"type": "array",
+					"items": {
+						"type": "number"
+					}
+				},
+				"associationGroupsMultiChannel": {
+					"type": "array",
+					"items": {
+						"type": "number"
+					}
+				},
+				"associationGroupsOptions": {
+					"type": "object",
+					"properties": {
+						"type": "object",
+						"properties": {
+							"hint": {
+								"$ref": "#/definitions/i18nObject"
+							}
+						}
+					}
+				},
+				"wakeUpInterval": {
+					"type": "number"
+				},
+				"defaultConfiguration": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"required": ["id", "size", "value"],
+						"properties": {
+							"id": {
+								"type": "number"
+							},
+							"size": {
+								"type": "number",
+								"enum": [
+									1,
+									2,
+									4
+								]
+							},
+							"value": {
+								"oneOf": [
+									{
+										"type": "number"
+									},
+									{
+										"type": "string"
```

Comment:
Is a string value for a default configuration a good idea? Personally I always use the numeric value, since I am not sure what happens when/if it is parsed.